lib/OpenQA/Setup.pm

```diff
@@ -121,6 +121,10 @@ sub read_config {
             screenshot_cleanup_batch_size             => OpenQA::Task::Job::Limit::DEFAULT_SCREENSHOTS_PER_BATCH,
             screenshot_cleanup_batches_per_minion_job => OpenQA::Task::Job::Limit::DEFAULT_BATCHES_PER_MINION_JOB,
         },
+	link_data => {
+            allow_types => ['schedule', 'autoyast_sle15'],
```

Comment:
again, if this is just for testing, ok. But openQA code itself knows nothing about SLE and should not know except for branding templates so something like `autoyast_sle15` should not be in the codebase.

Reply:
Yes, these configuration values should not end up in the code base but in the instance's config file. For OSD you can create a SR in our salt repo and on o3 I can edit the config manually for you if you don't have access.

---

I'm not sure whether you can use an array here. After all, this data structures are populated from an INI file. If it is not possible you can use a string and split it at white spaces.

---

Besides, I find the keys not well chosen. `link_data` and `allow_types` are not saying much what these values are used for. I'd use `job_settings_ui` and `keys_to_render_as_links`. It is of course a little bit longer but I hope the meaning is more self-explanatory this way.

Reply:
@okurz i go step by step as i 'discover' openQA and MOJO as i go and with your guidance. Next step is to put those values in a config file as @Martchus and @rwx788 have suggested